src/components/Search/SearchInput.tsx

```diff
@@ -35,13 +37,6 @@ const getMaxMenuHeight = (menuElement?: HTMLDivElement | null) => {
   return bodyHeight - menuTopOffset - 4;
 };
 
-type SearchItem = {
```

Comment:
Moved to `SearchTypes.ts` to prevent circular dependency in the new `SearchFeedback` component.